zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -504,9 +505,10 @@ def convert_slack_workspace_messages(slack_data_dir: str, users: List[ZerverFiel
                                      added_users: AddedUsersT, added_recipient: AddedRecipientsT,
                                      added_channels: AddedChannelsT, realm: ZerverFieldsT,
                                      zerver_realmemoji: List[ZerverFieldsT], domain_name: str,
-                                     output_dir: str, chunk_size: int=800) -> Tuple[List[ZerverFieldsT],
```

Comment:
Will this code run now? At least on the machine you're developing on.

Reply:
It won't, as the `MESSAGE_BATCH_CHUNK_SIZE` is set to `1000` in `/lib/export`. And as the gitter import has the same structure, it will fail. I was working fine with `800`.  

This is definitely a point of discussion. Which is what size should we set it as.